quantum/rgblight.c

```diff
@@ -1016,16 +1016,36 @@ void rgblight_effect_knight(animation_status_t *anim) {
 #endif
 
 #ifdef RGBLIGHT_EFFECT_CHRISTMAS
+#    ifndef RGBLIGHT_CHRISTMAS_EASING
+#        define RGBLIGHT_CHRISTMAS_EASING 3
```

Comment:
This setting should be added to the documentation, too.

Reply:
Sure, I can do that. However, I'm not entirely sure whether this should be a setting at all - it can only be 2 or 3 anyway, anything bigger results in an integer overflow. Also, 2 just doesn't look as good, so I think I'll completely remove that define (and replace it an a few other values I'm using with `consts` inside the function, to improve readability). What do you think?

Reply:
Okay, sounds good!